schema.json

```diff
@@ -223,6 +223,10 @@
         "table": {
           "description": "Name of table on which to define the index",
           "type": "string"
+        },
+        "predicate": {
```

Comment:
I'm wondering if this would be better called `where`, to avoid increasing cognitive load on the user

Reply:
I am open to changing it to anything that makes it clearer to the user.

Reply:
ah just saw this is actual postgres nomenclature in the docs, so we are good

Reply:
I think 'predicate' is OK as that is the term used in the Postgres [docs on partial indexes](https://www.postgresql.org/docs/current/indexes-partial.html).

I'm fine with either `where` or `predicate` personally.

Reply:
👍 let's use `predicate`